packages/core/admin/ee/server/routes/features-routes.js

```diff
@@ -43,4 +43,18 @@ module.exports = {
       },
     },
   ],
+  'review-workflows': [
```

Comment:
Same in the route names `/review-workflows/workflows`. My understanding would be that the feature is `review-workflow` and in this feature we are fetching `workflows`

Reply:
It depends.
The feature is `review-workflows` that use `workflows` and is described by a `workflow` content type.

Reply:
Make sense to me to have an `s` too.

Reply:
Yeah. We agreed we would use the plural internally because we will have multiple workflows soon and this helps us to avoid a big migration when we are there.

Reply:
Ok thanks!